lib/client-utils.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+const format = require('util').format;
+
+// this code is executed on client side, so we can not use features from es5
+exports.wrapLinkByTag = function(text) {
```

Comment:
`wrapLinkByTag` was moved to a separate file, so that `uglify` would not fail on ES6 classes in `lib/utils`.